phpBB/bin/phpbbcli.php

```diff
@@ -68,6 +68,11 @@
 $phpbb_container->get('request')->enable_super_globals();
 require($phpbb_root_path . 'includes/compatibility_globals.' . $phpEx);
 
+if (is_file($phpbb_root_path . $config['exts_composer_vendor_dir'] . '/autoload.php'))
+{
+	require_once($phpbb_root_path . $config['exts_composer_vendor_dir'] . '/autoload.php');
```

Comment:
I think it's better to ensure that what we put in the DB is validated than having to revalidate anything on usage yes (one reason being that this way the extensions doesn't have to revalidate the data they are using)